packages/renderer/src/lib/table/Table.svelte

```diff
@@ -40,6 +40,10 @@ function toggleAll(checked: boolean) {
 let sortCol: Column<any>;
 let sortAscending: boolean;
 
+$: if (data) {
+  sortImpl();
+}
+
```

Comment:
I would move the condition to the sort but it's only my feeling

Reply:
I would too, but it is using svelte reactivity here: sortImpl() is called whenever data changes to make sure the sort is maintained. e.g. if we get an update from provider that replaces all the objects the current sort is still applied.

Reply:
I mean, to do:

```
$: sortImpl();
```

and in sortImpl if we don't have data, we abort

Reply:
Yes, but if you remove `if (data)` then sortImpl() never gets called. Including data in the `$:` statement triggers svelte to register data as reactive and call sortImpl() whenever it gets changed.